docs/04-reference/wingsdk-spec.md

```diff
@@ -318,6 +359,15 @@ resource Bucket {
   inflight signed_url(key: str, duration?: duration): str;
 }
 
+struct FileMetadata {
+  /** The size of the file in bytes. */
+  size: Size;
+  /** The time the file was last modified. */
+  last_modified: Date; // or an ISO timestamp `str` until we have Date API support
+  /** The content type of the file, if it is known. */
+  content_type: str?;
```

Comment:
We could also return version number if versioning is enabled

Reply:
We should consider having semantic versions as a primitive in the language (`v12.0.1`)

Reply:
Good ideas. Since the spec doesn't mention versioning I'll leave it out for now but definitely sounds like a natural place to fit it in the future 👍